docs/Catalog/RemovingStacks.md

```diff
@@ -0,0 +1,100 @@
+that:
+
+- Your data in S3 buckets remains accessible.
+- You can continue using the Quilt SDK for reading and writing packages.
+- Stack-managed resources (Athena tables, user database, audit logs) will be affected.
+- The deletion behavior varies between CloudFormation and Terraform deployments.
+
+> **Consider Pausing Instead:** To temporarily suspend the stack, you can shut
+> down ECS services to reduce costs. Note that some charges may continue even
+> when ECS is inactive, so the only way to completely eliminate all costs is to
+> delete the stack.
+
+---
+
+## I. Impact Assessment and Preparation
+
+### What May Be Deleted
+
+- Custom configurations specific to the Quilt stack (e.g., IAM roles, resource
+  mappings, permissions).
+- Analytics data in stack-managed buckets.
+- Audit logs and Athena querying setups.
+- User database configurations.
+
+### Recommended Actions
+
+- Export analytics data you need to keep.
+- Save important audit logs.
+- Document existing Athena configurations if you'll need to recreate them.
+- If using Terraform, use it to capture a snapshot of the user database for
+  possible use in future stacks.
+
+---
+
+## II. Stack Deletion Process
+
+### Using CloudFormation
+
+1. **Start the Deletion Process**  
+   - Navigate to the **CloudFormation** console in your AWS region.  
+   - Select your stack under **CloudFormation > Stacks**.  
+   - Click **Delete Stack** to begin the process.
+
+2. **Empty or Ignore Non-Deleted Resources**
+   - CloudFormation attempts to delete all stack-managed resources.  
+   - If a resource cannot be deleted (e.g., non-empty S3 bucket):  
+     - It will show a **DELETE_FAILED** status in the **Events** tab.
+     - Failed resources remain intact, including non-empty S3 buckets.  
+     - Other independent resources will still be deleted.  
+     - The stack enters the **DELETE_FAILED** state.  
+   - If you wish to remove those resources:
+       - Back up any important files.  
+       - Remove all files from the bucket via the S3 console or CLI.  
+       - Retry the deletion process.
+   - Alternatively, you can leave them undeleted, though that may require manual
+     cleanup later.
+
+### Using Terraform
+
+1. **Start the Deletion Process**  
+   - Open a terminal in your Terraform configuration directory.  
+   - Run `terraform destroy`.
+
+2. **Empty or Retain Non-Deleted Resources**  
+   - Terraform stops at the first resource it cannot delete (e.g., non-empty S3
+     buckets).  
+   - Dependencies of the failed resource are preserved.  
+   - To resolve, back up important files and clear the bucket contents.  
+   - Alternatively, configure Terraform to retain the resource by marking it
+     for retention in the configuration.  For example:
+
+        ```hcl
+        resource "aws_s3_bucket" "logical_bucket_name" {
+          bucket = "actual-bucket-name"
+          lifecycle {
+            prevent_destroy = true
+          }
+        }
+        ```
```

Comment:
AFAICT the only bucket created directly by terraform (not via CloudFormation) is emptied automatically

Reply:
Meaning it WILL automatically empty the buckets it created, and will ONLY stop on "non-empty S3
     buckets created by CloudFormation"?

Reply:
yes